js-miniapp-bridge/src/bridge.js

```diff
@@ -15,22 +15,23 @@ var isPlatform = {
  * Method to call the native interface methods for respective platforms
  * such as iOS & Android
  * @param  {[String]} action Action command/interface name that native side need to execute
+ * @param  {String} param Action command/interface name that native side need to execute
```

Comment:
Still need to update this comment to `Object`.

Reply:
Ahh, thanks.